NGRValidator/NGRValidator/Categories/NSData+NGRValidator.h

```diff
@@ -0,0 +1,23 @@
+//
+//  NSData+NGRValidator.h
+//  NGRValidator
+//
+//
+//
+
+#import "NGRMimeType.h"
+
+@interface NSData (NGRValidator)
+
+NS_ASSUME_NONNULL_BEGIN
+
+- (BOOL)ngr_hasMimeType:(NGRMimeType *)type;
+
+- (BOOL)ngr_isImage;
```

Comment:
There are a lot of places like that throughout the project. Maybe it would be better to handle that as a part of another task like "Make API more swifty"?